third_party/ot-commissioner/build-android-libs.sh

```diff
@@ -39,10 +39,14 @@ find ./io/openthread/commissioner -name "*.class" | xargs jar cvf ../../../libs/
 cd ../../../
 
 ## Copy shared native libraries
-for lib in "$(find ./build -name "*.so")"; do
+<<<<<<< Updated upstream
```

Comment:
These conflict markers presumably should not be here...

Also, the commit messages for these commits really need to be better, please.

Reply:
@bzbarsky-apple Sorry for unintentional commits, reverted.